requirements-tests.txt

```diff
@@ -0,0 +1,11 @@
+-r requirements.txt
+pytest==3.0.5
```

Comment:
The pex build uses requirements.txt, so it probably is a good idea to keep test requirements separate.